libs/saveFiles.js

```diff
@@ -86,7 +93,7 @@ const attachFileToEntry = function (entry, fileDocument) {
 }
 
 const saveEntry = function (entry, options) {
-  if (!entry.fileurl && !entry.requestOptions) return entry
+  if (!entry.fileurl && !entry.requestOptions && !entry.filestream) return entry
```

Comment:
It starts to be unclear what it is the purpose of this check. I can be
clarified by introducing a function: 

```
const canBeSaved = entry => entry.fileurl || entry.requestOptions || entry.filestream

if (!canBeSaved(entry)) { return entry }
```